_includes/event-post-macros.html

```diff
@@ -0,0 +1,126 @@
+
+
+{# ==========================================================================
+
+   calendar_icon()
+
+   ==========================================================================
+
+   Description:
+
+   Create an HTML calendar icon when given:
+
+   date:           A valid date.
+
+   additional_classes: Extra classes you wish to add to the icon, space
+                       separated.
+
+   ========================================================================== #}
+
+{% macro calendar_icon(calendar_date, additional_classes) %}
+    <time class="calendar-icon float-media" data-month="{{ calendar_date | date('%b') }}" data-day="{{ calendar_date | date('%d') }}" datetime="{{ calendar_date }}">
```

Comment:
The float should probably be added outside of the macro, or added conditionally through a parameter, so the macro implementation can decide whether to float left or right or not at all (this is without looking into `float-media`—but I assume this adding a left float).